svelte-jest-legacy.js

```diff
@@ -4,5 +4,9 @@ module.exports = {
         'jest/globals': true
     },
     extends: ['./svelte-base-legacy', 'plugin:jest/recommended'],
-    plugins: ['jest']
+    plugins: ['jest'],
+    rules: {
+        'jest/expect-expect': ['off'],
```

Comment:
Off because we often like to place all related assertions into a helper function and this triggers the rule. [[docs](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md)]